dill/settings.py

```diff
@@ -9,20 +9,51 @@
 global settings for Pickler
 """
 
+__all__ = ['settings', 'Settings']
```

Comment:
I don't like having `settings` and `Settings`, and I'm not really convinced that we need the changes here.  Why should we use `Settings` instead of a simple dict?

Reply:
reverted